src/commands/prepare.ts

```diff
@@ -133,15 +130,15 @@ This command prepares a Provider JSON metadata definition that can be used to ge
       (providerJson.services.length === 1 &&
         providerJson.services[0].baseUrl.includes('TODO'))
     ) {
-      // TODO: provide more info - url to docs
+      // TODO: provide more info - url to REAL docs
       ux.warn(
-        `Provider definition prepared, but some parts are missing. Please fill them manually in ${providerJsonPath}.`
+        `Provider definition prepared, but some parts are missing. Please fill them manually in ${providerJsonPath}. Link to documentation:\nhttps://pr-115-superface-docs.vercel.app/docs/guides/editing-provider-files\nYou can use it to generate integration code interface with 'superface new ${providerJson.name} "<use case description>"'.`
```

Comment:
I made a better link using our shortener. You can use it now, we'll upgrade it to production link once the documentation is deployed:
https://sfc.is/editing-providers